requirements.txt

```diff
@@ -4,5 +4,5 @@ wheel
 Cython
 Sphinx
 flake8
-tomli
+tomli; python_version < '3.11'
```

Comment:
I don't get why it says `tomli` rather than `tomlli`. (or it should be `tomli` in `conf.py`, not `tomlli`